styles.css

```diff
@@ -186,3 +187,79 @@ ul {
 .crcl {
   padding: 2px;
 }
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  width: 375px;
+  margin: auto;
+  background-color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  background-image: url(/images/bglmnt2.png);
```

Comment:
- It appears you have a tiny bug here and this is causing your `background-image` file not to show/render. See 👇 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/213033804-859d6044-e11d-4b6d-83c9-b889f81029d9.png)|![image](https://user-images.githubusercontent.com/99715640/213033882-7880cf0d-d86c-455b-b54f-cfd00f7a687c.png)|

- [ ] I think inserting a period `.` before other file paths will resolve this issue. You might be able to see this from your browser and that's because the image is on your local machine, and also the influence of `cache` memory. The background image would really go extra mile to make your design more good looking than it is now.
- [ ] Please try this instead:
```
background-image: url(./images/bglmnt2.png);
```
- [ ] To further improve your design with breath-taking user experience, please try to check the values of your button interactivity as it is not as expected. Here 👇 is a clip of what the interactivity states look like: 

![image](https://user-images.githubusercontent.com/99715640/213034806-c3d47aa8-fbe9-44f5-a4cc-75ba40f82fd4.png)

- [ ] As you may have notice, that there is a separating element giving the contact section a more aesthetic feeling 👇 
![image](https://user-images.githubusercontent.com/99715640/213035064-5d13b400-d97e-4d07-befd-01906245faa2.png)

How about about adding it to your page too? I think it will be cool, and thankfully, it is very easy to implement. 😉